open_academy/models/session.py

```diff
@@ -11,6 +11,7 @@ class Session(models.Model):
     duration = fields.Float()
     seats = fields.Integer(string="Number of seats")
     percent_seats = fields.Integer(string="Taken seats %", compute="_compute_seats")
+    attendee_count = fields.Integer(compute="_compute_seats", store=True)
```

Comment:
You are using the same compute method for percent_seats and attendee_count that depends on the seats and attendee_id. 
Why not add another compute method that only depends on the attendee_ids since is the only value that I think matters to compute the values? 
@frahikLV What do you think?

Reply:
Yes, I agree, this will prevent it from triggering and calculating multiple times the value for this field.

Reply:
Done